### DISCUSSION TITLE
Building in unclaimed land with grief prevention mod

### DISCUSSION THREAD

**Discussion by pgiovanni** [GriefPrevention/GriefPrevention#1396] [Q&A] [answered]
I want to let players build in unclaimed land but still have the grief prevention mod enabled for those who want claims. How can I do this? Right now players cannot build in unclaimed land.

**Comment by bigpresh** [comment, marked as answer]
Set claims.mode to "Survival" rather than "Creative" or "SurvivalRequiringClaims".

The documentation on that option could be improved a little actually ~~- I'll do that now~~.

**Reply by pgiovanni** [reply]
Thanks that worked! it also solved my issue of being able to drop items

**Reply by RoboMWM** [reply]
if you want to make an edit just add the raw markdown here or whatever with what you want to change.
